### PR TITLE
Readme & API RH & PyTest

### LINKED ISSUE #95: API RedHat Jobs

### PR DESCRIPTION
**Pull request by RomTamam**
Fix #95

### CODE REVIEW

File: tests/tests_API_FB.py [hunk 1]
```diff
@@ -9,21 +9,52 @@ def test_api_urls():
...
 
 
 def test_api_titles():
     # Create soup object & print a status message - 404 not good , 200 good
     soup = API_FB.create_soup(API_FB.create_req())
     """The testing will check if our functions return the required output"""
-    expected_output_titles = ['Production Engineer', 'Offensive Security Engineer Intern, Red Team',
-                              'Internal Audit Manager – Infrastructure', 'Production Engineer',
-                              'Production Engineer', 'Production Engineer', 'Production Engineer']
+    expected_output_titles = ['Engineering Manager - Malware Analysis Infrastructure',
```
Comment (simzacks) [inline tests/tests_API_FB.py:18]: Tests failed. see Checks tab above

**Review by simzacks — CHANGES_REQUESTED** [review]
Generally it would be wise to separate the readme, API_RH and API_FB into 3 different PRs.
However, we don't have a lot of time left, so if you get the tests working and don't add any other code, I'll approve it

File: tests/tests_API_FB.py (hunk 1 above)
Comment (RomTamam) [reply tests/tests_API_FB.py:18]: Thanks for your feedback
As you mentioned, we're running out of time and I think we're all good with those changes
I'll appreciate your review/approve

File: tests/API_FB.py [hunk 2]
```diff
@@ -34,12 +34,16 @@ def extract_jobs_title(soup):
...
 
 def create_soup(req):
-    print(req.status_code)
     soup = BeautifulSoup(req.content, 'html.parser')
     # print(soup.prettify())
     return soup
 
 
+def check_status_code(req):
+    print(req.status_code)
```
Comment (simzacks) [inline tests/API_FB.py:43]: What is the purpose of this function as opposed to just using req.status_code directly?

File: tests/API_RH.py [hunk 3]
```diff
@@ -34,12 +34,16 @@ def extract_jobs_title(soup):
...
 
 def create_soup(req):
-    print(req.status_code)
     soup = BeautifulSoup(req.content, 'html.parser')
     # print(soup.prettify())
     return soup
 
 
+def check_status_code(req):
+    print(req.status_code)
```
Comment (simzacks) [inline tests/API_RH.py:43]: What is the purpose of this function as opposed to just using req.status_code directly?

File: tests/API_FB.py (hunk 2 above)
Comment (RomTamam) [reply tests/API_FB.py:43]: for the PyTests, I chagnes the URL testing to valid URL instead of static URLs..

File: tests/API_RH.py (hunk 3 above)
Comment (RomTamam) [reply tests/API_RH.py:43]: same answer as above

File: tests/API_FB.py (hunk 2 above)
Comment (simzacks) [reply tests/API_FB.py:43]: The only thing that this function does is print the status_code (which you don't need to do) and return it.
In the tests function for example:
`assert API_FB.check_status_code(API_FB.create_req()) == 200, "The tests for URLs were successful"`

It should be 
`assert API_FB.create_req().status_code == 200, "The tests for URLs were successful"`

Comment (RomTamam) [reply tests/API_FB.py:43]: Just saw your review, I'll fix it

**Pull request closed without merging**